addons/website_forum/populate/forum_post.py

```diff
@@ -0,0 +1,42 @@
+from random import choices
+from odoo import models
+from odoo.tools import populate
+
+QUESTION_SIZES = { 'small': 100, 'medium': 1000, 'large': 90000 }
+MAX_REPLY_NUM = 7
+POST_SIZES = { key: size * (MAX_REPLY_NUM + 1) for key, size in QUESTION_SIZES.items() }
+QA_WEIGHTS = { '0': 25, '1': 35, '2': 20, '3': 10, '4': 4, '5': 3, '6':2, '7': 1 }
+ANSWERS_RANGE = list(range(0, (MAX_REPLY_NUM + 1)))
+WEIGHTS = list(QA_WEIGHTS.values())
```

Comment:
These are many lines all turning around the weights and the number of replies, but the weights is not computed based on MAX_REPLY_NUM, (and I don't think we want to change it that often) so I'm not sure this is worth it compared to something shorter like this: 


```suggestion
QUESTION_SIZES = {'small': 100, 'medium': 1000, 'large': 90000}
POST_SIZES = {key: size * 3 for key, size in QUESTION_SIZES.items()}
QA_WEIGHTS = {0: 25, 1: 35, 2: 20, 3: 10, 4: 4, 5: 3, 6: 2, 7: 1}  # Mean of 2.
```
The same applies for messages of course.

Also note that with your weights distribution, the mean and median values of answers-to-posts is 2, which so `3` is a better multiplier for POST_SIZES, right?